src/components/modal/ModalDialog.js

```diff
@@ -99,6 +132,7 @@ class ModalDialog extends React.Component<Props> {
             modalRef,
             title,
             type,
+            showBackButton,
```

Comment:
Can we use the existence of `onBackClick` to manage this condition, similar to the check in `renderBackButton`?

Reply:
We need the showBackButton because when we create the Modal, we can hide/show the back button depending on the current screen...The callback function used on renderBackButton will be defined regardless of showing/hiding the back button.